spec/classes/web_spec.rb

```diff
@@ -56,7 +56,20 @@ def package_provider_for_gems
           let :params do
             super().merge(database_type: 'postgresql')
           end
-          packages = facts[:osfamily] == 'RedHat' ? ['zabbix-web-pgsql', 'zabbix-web'] : ['zabbix-frontend-php', 'php5-pgsql']
+
+          pgsqlpackage = ""
+          case facts[:osfamily]
+          when 'Debian'
+            if facts[:operatingsystemmajrelease] >= '16.04'
+              pgsqlpackage = "php-pgsql"
```

Comment:
migrate all of the double quotes to single quotes unless they contain a variable